flask_project/campaign_manager/test/test_survey.py

```diff
@@ -15,7 +15,7 @@ def test_init_survey_buildings(self):
             'buildings.yml'
         )
         survey = Survey(surveyPath)
-        self.assertEqual(survey.data['tags']['building'], ['yes'])
+        self.assertEqual(survey.data['feature'], "building=yes")
```

Comment:
@russbiggs @JorgeMartinezG I'm not sure why the templates on s3 would have created an object where `survey.data['tags']['building']` is `['yes']`. The YAML doesn't have a tag named `building` with the value `['yes']`. But this test was passing with that check before.

The test seems to be intended just to test that the survey object is created, so I changed the test to check that `survey.data['feature']` is `"building=yes"` instead. The tests passes, but I just wanted to point this out in case either of you think the result of this function should be different.

Reply:
Determined that this was because the test file was testing against a fixture with a different structure than the new .yml files that have been added.